news/index.md

```diff
@@ -57,9 +57,9 @@ which also contains links to the Slack channel and mailing lists.
 
 ### The OCFL Editors
 
-Andrew Hankinson (Bodleian Libraries, University of Oxford)
-Neil Jefferies (Bodleian Libraries, University of Oxford)
-Rosalyn Metz (Emory University)
-Julian Morley (Stanford University)
-Simeon Warner (Cornell University)
+Andrew Hankinson (Bodleian Libraries, University of Oxford)  
```

Comment:
This may cause a problem later for those of us who use editors set to strip trailing whitespace on save...

Reply:
Maybe a bulleted list is the compromise:
```
- Andrew Hankinson (Bodleian Libraries, University of Oxford)
- Neil Jefferies (Bodleian Libraries, University of Oxford)
- Rosalyn Metz (Emory University)
- Julian Morley (Stanford University)
- Simeon Warner (Cornell University)
- Andrew Woods (LYRASIS)
```

Reply:
There are other methods markdown uses. Will update after dinner.